openedx/features/caliper_tracking/tests/expected/edx.drag_and_drop_v2.feedback.closed.json

```diff
@@ -0,0 +1,47 @@
+{
+  "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "action": "ClosedPopout",
```

Comment:
`ClosedPopout` is specific to video poputs. Please use another `action`. `Viewed` seems appropriate.

Reply:
done